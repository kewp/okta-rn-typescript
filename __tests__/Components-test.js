/* leave first line blank for cq */
import Page from '../components/Page';
import Form from '../components/Form';
import Heading from '../components/Heading';

import React from 'react';
import renderer from 'react-test-renderer';

test('Page renders correctly', () => {
  const tree = renderer.create(<Page />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Form renders correctly', () => {
	const tree = renderer.create(<Form />).toJSON();
	expect(tree).toMatchSnapshot();
});

test('Heading renders correctly', () => {
	const tree = renderer.create(<Heading />).toJSON();
	expect(tree).toMatchSnapshot();
});