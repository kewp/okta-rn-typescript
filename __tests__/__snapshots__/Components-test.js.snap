// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
/>
`;

exports[`Heading renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "color": "black",
        "fontFamily": "GillSans-light",
        "fontSize": 32,
        "marginTop": 120,
        "textAlign": "center",
      },
    ]
  }
/>
`;

exports[`Page renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 40,
      },
    ]
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/background.jpg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
</View>
`;
